python/foglamp/services/south/ingest.py

```diff
@@ -173,13 +173,16 @@ class Ingest(object):
             "key": category,
             "description": 'South Service configuration',
             "value": default_config,
-            "keep_original_items": False
+            "keep_original_items": True
```

Comment:
Having this keep_original_items in the JSON is now obsolete, it is a parameter. Why has this changed from False to True, I would have expected the same behaviour here?